commons-search/src/main/resources/es-default-index-settings.json

```
@@ -27,6 +27,13 @@
                "lowercase",
                "asciifolding"
             ]
+         },
+         "keyword_html_strip":{
+            "type":"custom",
+            "tokenizer": "keyword",
+            "char_filter": [
+               "html_strip"
+            ]
```

It should extend `whitespace_lowercase_asciifolding` by adding `html_strip`

```suggestion
         },
         "whitespace_lowercase_asciifolding_html_strip":{
            "type":"custom",
            "tokenizer":"whitespace",
            "filter":[
               "lowercase",
               "asciifolding"
            ],
            "char_filter": [
               "html_strip"
            ]
```